ui/document_uploader.py

```diff
@@ -564,72 +542,91 @@ def upload_scanned_certs(self, scanned_certs):
+                        u'Error creating file: {0}'.format(
+                            '`' + new_filename + '`'
+                        )
+                    )
+                    self.upload_progress.emit(
+                        UploadWorker.ERROR,
+                        msg
+                    )
                     continue
 
-                msg = _translate(
-                    'DocumentUploader',
-                    u'Finished creating file: {0}',
-                    None
-                    ).format( '`' + new_filename + '`')
-                self.upload_progress.emit(UploadWorker.INFORMATION, msg)
+                msg = self.tr(
+                    u'Finished creating file: {0}'.format(
+                        '`' + new_filename + '`'
+                    )
+                )
+                self.upload_progress.emit(
+                    UploadWorker.INFORMATION,
+                    msg
+                )
                 #Delete After Upload
                 if self.del_after_upload:
                     try:
                         old_filename=scanned_cert['full_filename']
                         os.remove(old_filename)
-                        msg = "Removed file: `"+old_filename+"`"
-                        self.upload_progress.emit(UploadWorker.INFORMATION, msg)
+                        msg = 'Removed file: `'+ old_filename + '`'
+                        self.upload_progress.emit(
+                            UploadWorker.INFORMATION,
+                            msg
+                        )
                     except:
-                        msg = "Failed to Removed file: `"+old_filename+"`"
-                        self.upload_progress.emit(UploadWorker.ERRORINFORMATION, msg)
+                        msg = ('Failed to Removed file: `'
+                            + old_filename
+                            + '`')
+                        self.upload_progress.emit(
+                            UploadWorker.ERRORINFORMATION,
+                            msg
+                        )
 
 
                 # Delete After Upload
                 if self.del_after_upload:
                     old_filename = scanned_cert['full_filename']
                     try:
                         os.remove(old_filename)
-                        msg =  _translate(
-                            'DocumentUploader',
-                            u'Removed file: {0}',
-                            None
-                            ).format('`' + old_filename + '`')
+                        msg =  self.tr(
+                            u'Removed file: {0}'.format(
+                                '`' + old_filename + '`'
+                            )
+                        )
                         self.upload_progress.emit(
                             UploadWorker.INFORMATION,
                             msg
                         )
                     except:
                         pass

```

Comment:
The exception will go without being noticed.

Reply:
Dear @pgathogo ,
Kindly be informed that after try-except there is a test is file is still exists and a message in case if the file is exists
>           if os.path.exists(old_filename):
>              msg = self.tr('Failed to Removed file: {0}'.format('`' + old_filename + '`'))
>               self.upload_progress.emit(UploadWorker.ERRORINFORMATION,msg)

Kindly be informed I moved this block to commands within except block